tests/admin-unlock/main.yml

```diff
@@ -0,0 +1,316 @@
+      when: g_rpm_name is undefined or
+            g_rpm_name_2 is undefined
+
+  roles:
+    # Subscribe if the system is RHEL
+    - role: redhat_subscription
+      when: ansible_distribution == 'RedHat'
+      tags:
+        - redhat_subscription
+
+- name: Ostree Admin Unlock - Add/Remove Package - Non-persistence
+  hosts: all
+  become: yes
+
+  tags:
+    - unlock_add_remove_package_non_persistence
+
+  vars_files:
+    - vars.yml
+
+  pre_tasks:
+    - name: set rpm_name
+      set_fact:
+        rpm_name: "{{ g_rpm_name }}"
+
+  roles:
+    - role: ostree_admin_unlock
+
+    - role: overlayfs_verify_present
+
+    - role: rpm_install
+
+    - role: package_verify_present
+
+    - role: rpm_uninstall
+
+    - role: package_verify_missing
+
+    - role: rpm_install
+      tags:
+        - second_install
+
+    - role: package_verify_present
+      tags:
+        - second_verify_present
+
+    - role: reboot
+
+    - role: overlayfs_verify_missing
+
+    - role: package_verify_missing
+      tags:
+        - second_verify_missing
+
+- name: Ostree Admin Unlock - Hotfix - Add/Remove Package Persistence
+  hosts: all
+  become: yes
+
+  tags:
+    - hotfix_add_remove_package_persistence
+
+  vars_files:
+    - vars.yml
+
+  pre_tasks:
+    - name: set rpm_name
+      set_fact:
+        rpm_name: "{{ g_rpm_name }}"
+
+  roles:
+    - role: ostree_admin_unlock_hotfix
+
+    - role: overlayfs_verify_present
+
+    - role: rpm_install
+
+    - role: package_verify_present
+
+    - role: rpm_uninstall
+
+    - role: package_verify_missing
+
+    - role: rpm_install
+      tags:
+        - second_install
+
+    - role: package_verify_present
+      tags:
+        - second_verify_present
+
+    - role: reboot
+
+    - role: overlayfs_verify_present
+
+    - role: package_verify_present
+      tags:
+        - third_verify_present
+
+    - role: rpm_install
+      rpm_name: "{{ g_rpm_name_2 }}"
+      tags:
+        - third_install
+
+    - role: package_verify_present
+      rpm_name: "{{ g_rpm_name_2 }}"
+      tags:
+        - fourth_verify_present
+
+- name: Ostree Admin Unlock - Hotfix - Rollback
+  hosts: all
+  become: yes
+
+  tags:
+    - hotfix_rollback
+
+  vars_files:
+    - vars.yml
+
+  pre_tasks:
+    - name: set rpm_name
+      set_fact:
+        rpm_name: "{{ g_rpm_name }}"
+
+  roles:
+    - role: rpm_ostree_rollback
+
+    - role: reboot
+
+    - role: overlayfs_verify_missing
+
+    - role: package_verify_missing
+
+    - role: package_verify_missing
+      rpm_name: "{{ g_rpm_name_2 }}"
+      tags:
+        - second_package_verify_missing
+
+- name: Ostree Admin Unlock - No Install Without Unlock
+  hosts: all
+  become: yes
+
+  tags:
+    - cannot_install_without_unlock
+
+  vars_files:
+    - vars.yml
+
+  tasks:
+    - name: set rpm_name
+      set_fact:
+        rpm_name: "{{ g_rpm_name }}"
+
+    - name: Get one of the rpms that was downloaded
+      shell: ls /root | grep {{ rpm_name }}*
+      register: rpm
```

Comment:
That's an assumption that is made because we don't technically restore to a clean state between each test. I was under the impression that separating the plays was to produce easier to read test logs not necessarily to separate the tests.  Do you think it is better to duplicate a bunch of tasks or just add this into the README.md?